be/src/olap/rowset/segment_v2/inverted_index_reader.cpp

```diff
@@ -353,7 +353,7 @@ Status FullTextIndexReader::query(OlapReaderStatistics* stats, RuntimeState* run
         }
 
         std::unique_ptr<lucene::search::Query> query;
-        query_info.field_name = std::wstring(column_name.begin(), column_name.end());
```

Comment:
grep std::wstring in the code and fix all

Reply:
done